react-nd/src/consts/Routes.jsx

```diff
@@ -0,0 +1,36 @@
+import Home from '@/views/home/Home';

```

Comment:
Šis failas iš didžiosios raidės, kityi iš mažųjų. Suderinkite, kad visur būtų vienodai